src/GoTrueApi.ts

```diff
@@ -203,6 +203,34 @@ export default class GoTrueApi {
     }
   }
 
+  /**
+   * Logs in an OpenID Connect user using their id_token.
+   * @param id_token The id_token of the user.
+   * @param nonce The nonce of the user.
+   * @param provider The provider of the user.
+   */
+  async signInWithIDToken(
```

Comment:
It will be OpenID Connect, not OpenID.

However, signInwithOpenIDConnect seems to be too long.

Also, OIDC is often used as an abbreviation for OpenID Connect.
If you do a Google search for "OIDC", OpenID Connect is the one that comes up at the top, and there are no other confusing options.

What about signInwithOIDC?